Packs/BluelivThreatContext/Integrations/BluelivThreatContext/BluelivThreatContext.py

```diff
@@ -908,12 +935,12 @@ def blueliv_signature(client: Client, signatureName, signatureId):
                 'HumanReadable': tableToMarkdown("Blueliv Signature info", human),
                 'ReadableContentsFormat': formats['markdown'],
                 'EntryContext': {
-                    'threatContext.hasResults': 'true',
-                    'signature.name': name,
-                    'signature.type': signatureType,
-                    'signature.updatedAt': updatedAt,
-                    'signature.malware': malware,
-                    'signature.malwareIds': malwareIds
+                    'BluelivThreatContext.threatContext.hasResults': 'true',
+                    'BluelivThreatContext.signature.name': name,
+                    'BluelivThreatContext.signature.type': signatureType,
+                    'BluelivThreatContext.signature.updatedAt': updatedAt,
+                    'BluelivThreatContext.signature.malware': malware,
+                    'BluelivThreatContext.signature.malwareIds': malwareIds
```

Comment:
assuming that a signature name is a unique field.
also, why do you need this path `BluelivThreatContext.threatContext.hasResults`? you didn't declared it as a command output (for any command)
```suggestion
                    'BluelivThreatContext.signature(val.name && val.name == obj.name)': {
                        'name': name,
                        'type': signatureType,
                        'updatedAt': updatedAt,
                        'malware': malware,
                        'malwareIds': malwareIds,
                    }
```

same can be done for all context outputs.